run.py

```
@@ -0,0 +1,52 @@
+
+    port = 8000  # Use port 8000 to avoid macOS AirPlay conflict
+    print("🚀 Starting Source2Social Flask application...")
+    print(f"🌐 Open http://localhost:{port} in your browser")
+    print("📝 Dashboard will show your generated posts")
+    print(f"🔗 Webhook endpoint: http://localhost:{port}/webhook/github")
+    print()
+    print("💡 To test the webhook, run: python test_webhook.py")
+    print()
+
+    # Run the Flask application
+    app.run(debug=True, host="0.0.0.0", port=port)
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)